renderer.ts

```diff
@@ -70,6 +70,10 @@ copySync(IN_FOLDER, OUT_FOLDER, {
   filter(src: string, dest: string): boolean { return src != TEMPLATE_FILE; }
 })
 
+//create the directory for podcast episodes, so that the build doesn't fail silently
```

Comment:
This should probably get moved over to the deploy.sh script at some point, as it is specific to how we serve the site rather than how we build it.